selection_sort/Python/selection_sort.py

```diff
@@ -0,0 +1,28 @@
+'''
+ ____       _           _   _             ____             _   
+/ ___|  ___| | ___  ___| |_(_) ___  _ __ / ___|  ___  _ __| |_ 
+\___ \ / _ | |/ _ \/ __| __| |/ _ \| '_ \\___ \ / _ \| '__| __|
+ ___) |  __| |  __| (__| |_| | (_) | | |  ___) | (_) | |  | |_ 
+|____/ \___|_|\___|\___|\__|_|\___/|_| |_|____/ \___/|_|   \__|
+
+'''
+# author: john2ksonn
+# date: 10.28.2018
+
+import random
+
+def sort(a):
+    max_idx = len(a) - 1
+    for idx in range(0, max_idx):
+        min_idx = idx        
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpz6y7xvz3/selection_sort/Python/selection_sort.py
+++ b/tmp/tmpz6y7xvz3/selection_sort/Python/selection_sort.py
@@ -14,7 +14,7 @@
 def sort(a):
     max_idx = len(a) - 1
     for idx in range(0, max_idx):
-        min_idx = idx        
+        min_idx = idx
         for i in range(idx+1, max_idx+1):
             if a[i] < a[min_idx]:
                 min_idx = i
```